distributed/dashboard/components/scheduler.py

```diff
@@ -1971,12 +1971,12 @@ def convert(self, msgs):
     @without_property_validation
     @log_errors
     def update(self):
-        log = self.scheduler.get_events(topic="stealing")
-        current = len(self.scheduler.events["stealing"])
-        n = current - self.last
-
-        log = [log[-i][1][1] for i in range(1, n + 1) if log[-i][1][0] == "request"]
-        self.last = current
+        topic = self.scheduler._broker._topics["stealing"]
```

Comment:
I don't like hooking deeply into these private attributes but I don't care enough for now to do something about it.

Reply:
I'm not even sure if we need this event but let's postpone this decision